[DOC] Update Dev Tools for navigation changes (#4874)
opensearch-project/documentation-website: _dashboards/dev-tools/index-dev.md

# Dev Tools

> Interact directly with OpenSearch by using **Dev Tools** to set up your OpenSearch Dashboards environment, run queries, explore data, and debug problems. You can use the Dev Tools console to: 

Reviewer: ```suggestion
**Dev Tools** is a development environment that lets you set up your OpenSearch Dashboards environment, run queries, explore data, and debug problems. You can use the Dev Tools console to: 
```

- **Set up your OpenSearch Dashboards environment.** For example, you can use the console to configure authentication settings for your OpenSearch Dashboards instance. 

> - **[Run queries to explore your data]({{site.url}}{{site.baseurl}}/dashboards/dev-tools/run-queries/).** For example, you can use the console to run a query to find all the documents in your index that contain a specific word.

Reviewer: ```suggestion
- **[Run queries to explore your data]({{site.url}}{{site.baseurl}}/dashboards/dev-tools/run-queries/).** For example, you can use the console to tune your queries for relevance.
```

> - **Debug problems with your queries.** For example, if your query is not returning the results you expect, you can use the console to look for error messages and identify the problem.

Reviewer: ```suggestion
- **Debug problems with your queries.** For example, if your query is not returning the results you expect, you can use the console to identify the problem.
```

> - **Learn about the APIs in OpenSearch.** For example, you can use the API reference documentation linked in the console (select the question circle icon ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/question-circle.png" class="inline-icon" alt="question circle icon"/>{:/})) to look up the syntax for different API calls.

Reviewer: ```suggestion
- **Learn about the APIs in OpenSearch.** For example, you can use the API reference documentation linked in the console to look up the syntax for different API calls (select the question circle icon ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/question-circle.png" class="inline-icon" alt="question circle icon"/>{:/})).
```

> - **Develop custom visualizations.** For example, you can use the console to create Vega visualizations. 

Reviewer: Should we add a link to Vega visualizations docs here?

- **Customize the appearance and behavior of dashboards.** For example, you can use the console to customize dashboard visualization colors or to add new filters. 

> - **Identify and fix bugs.** For example, you can use the console to view logs and identify the cause of the problem.

Reviewer: I'm not sure I understand what this bullet point means. What kinds of logs? Dashboards logs or user custom logs? And bugs where?

Author: Deleted. If dev provides more details, I'll add later.
